docker/env/hydra.sh

```diff
@@ -57,7 +57,7 @@ docker run --rm ${TTY_STDIN} --privileged \
     -w ${WORK_DIR} \
     -e JOB_NAME=${JOB_NAME} \
     -e BUILD_URL=${BUILD_URL} \
-    -u $(id -u ${USER}):$(id -g ${USER}) \
+    -u 1000:999 \
```

Comment:
are you sure you wanted to change this ?

Reply:
No. will return back

Reply:
a small reminder, seem you didn't changed this back yet